tests/test_routes.py

```diff
@@ -205,6 +205,7 @@ def test_route_sub_criteria_scoring(
                 soup.title.string
                 == "Score - Engagement - Community Gym - Assessment Hub"
             )
+            assert b"This is a comment" in response.data
```

Comment:
Not part of this PR but it reminds me of @srh-sloan [post](https://communities-govuk.slack.com/archives/C02PHBER287/p1675866903115839) about testing templates, this looks like another approach 
[here](https://github.com/communitiesuk/funding-service-design-assessment/blob/1b63c296082702b2a73361d5d23b1a00d221529b/tests/test_routes.py#L645:~:text=assert%201%20%3D%3D-,len,-(templates_rendered))

Reply:
I'm creating a page for standardised testing approaches that people can use to guide them in writing tests on our repos going forward, maybe we can out a chat in next week with others and put something concrete in the guidance?